consensus/misc/eip4844/eip4844.go

```diff
@@ -87,6 +87,8 @@ func CalcBlobFee(config *params.ChainConfig, header *types.Header) *big.Int {
 		frac = config.BlobScheduleConfig.Prague.UpdateFraction
 	case forks.Cancun:
 		frac = config.BlobScheduleConfig.Cancun.UpdateFraction
+	case forks.Osaka:
```

Comment:
nitpick, can we put Osaka before the Prague with the order from latest fork to oldest one?

Reply:
Yep sry, I always forget the ordering between cancun and prague